server/lib/nl/page_config_next.py

```diff
@@ -264,10 +267,13 @@ def _multiple_place_bar_block(column, places: List[Place], svs: List[str],
     title = attr['title']
   elif len(svs) > 1:
     # This suggests we are comparing against SV peers from SV extension
-    title = 'Compare with Other Variables'
+    title = 'Comparison with Other Variables'
```

Comment:
How about "Compared with"? Otherwise should be "In comparison with"

Reply:
Done.